### ISSUE TITLE
Apical cap

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6686]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
An apical cap is a caplike lesion at the lung apex, usually caused by intrapulmonary and pleural fibrosis pulling down extrapleural fat or possibly by chronic ischemia resulting in hyaline plaque formation on the visceral pleura. The prevalence increases with age. It can also be seen in hematoma resulting from aortic rupture or in other fluid collection associated with infection or tumor, either outside the parietal pleura or loculated within the pleural space. The usual appearance is of homogeneous soft-tissue attenuation capping the extreme lung apex (uni- or bilaterally), with a sharp or irregular lower border. Thickness is variable, ranging up to about 30 mm. An apical cap occasionally mimics apical consolidation on transverse CT scans.

PMID:18195376

Comment
See Figure 6 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
